package.json

```diff
@@ -47,8 +47,11 @@
   },
   "dependencies": {
     "backbone": "^1.3.3",
+    "file-loader": "^4.0.0",
+    "font-awesome": "^4.7.0",
     "jade": "^1.11.0",
     "jquery": "^3.1.1",
+    "source-sans-pro": "^2.45.0",
```

Comment:
This is really cool, that we can just include the font directly from npm.

Reply:
I am honestly like more excited about that than almost anything else in this PR 😁